### PR TITLE
docs: fix fireEvent Example code

### PR DESCRIPTION
**Pull request by pppp606**
When I copied the code example, there was an error, so I fixed it.
https://testing-library.com/docs/preact-testing-library/api/#example-1

- Fix syntax error
- Delete undefined prop
  - https://developer.mozilla.org/en-US/docs/Web/API/Event/Event

### CODE REVIEW

File: docs/preact-testing-library/api.mdx [hunk 1]
```diff
@@ -113,12 +113,12 @@ fireEvent.click(buttonNode);
...
-buttonNode,
-new Event('MouseEvent', {
-  bubbles: true,
-  cancelable: true,
-  button: 0,
-});
+  buttonNode,
+  new Event('MouseEvent', {
+    bubbles: true,
+    cancelable: true,
```
Comment (timdeschryver) [inline docs/preact-testing-library/api.mdx:119]: Thanks for the fix, though I would like to re-add the `button` property.

```suggestion
    cancelable: true,
    button: 0
```

`button` is a valid event property - https://developer.mozilla.org/en-US/docs/Web/API/MouseEvent/button - and is also impolemented like this in fireEvent - https://github.com/testing-library/dom-testing-library/blob/main/src/event-map.js#L82.

Comment (ph-fritsche) [reply docs/preact-testing-library/api.mdx:119]: @pppp606 is correct. There is no `button` property on `Event`. It is implemented on `MouseEvent`.
https://codesandbox.io/s/event-property-button-p6w8xh?file=/src/index.test.js

Comment (ph-fritsche) [inline docs/preact-testing-library/api.mdx:119]:
```suggestion
  new MouseEvent('click', {
    bubbles: true,
    cancelable: true,
    button: 0,
```

**Comment by pppp606** [comment]
@timdeschryver @ph-fritsche 
Thanks to both of you. I have committed the suggestion, please recheck pr.

**Review by timdeschryver — APPROVED** [review]

**Comment by timdeschryver** [comment]
@all-contributors please add @pppp606 for docs

**Pull request merged**